src/main/kotlin/no/nb/mlt/wls/application/hostapi/item/ApiItemPayload.kt

```diff
@@ -73,15 +72,13 @@ data class ApiItemPayload(
     @Schema(
         description = """Where the item is located, can be used for tracking item movement through storage systems.""",
         examples = ["SYNQ_WAREHOUSE", "AUTOSTORE", "KARDEX"],
-        accessMode = READ_ONLY,
         required = false
     )
     val location: String?,
```

Comment:
Can probably mark these as non-nullable
```suggestion
    val location: String,
```